apps/backoffice-v2/src/pages/Entity/components/EditableDetails/helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { AnyObject } from '@ballerine/ui';
+
+export const isNullishDateValue = <T = unknown>(value: T, jsonSchema: AnyObject): boolean => {
+  if (!value && jsonSchema?.format === 'date') return true;
```

Comment:
We have a utility called `isNullish`. Nullish should only cover `undefined` and `null`. 🙂